sql/organization_docs/miri-org-docs.sql

```diff
@@ -216,9 +216,9 @@ insert into organization_documents(url, title, publication_date, modified_date,
         'Effective Altruism Forum', /* publisher */
         'Machine Intelligence Research Institute', /* affected_organizations */
         NULL, /* affected_people */
-        NULL, /* document_scope */
+        'Hiring-related notice', /* document_scope */
         'AI safety', /* cause_area */
-        'In response to a question, Soares writes that MIRI has decided against hiriting senior math people in a supervisor role, and also writes that MIRI is bottlenecked on technical writing ability.' /* notes */
+        'In response to a question, Soares writes that MIRI has decided against hiriing senior math people in a supervisor role, and also writes that MIRI is bottlenecked on technical writing ability.' /* notes */
```

Comment:
Thanks for correcting the typo, but it looks like your corrected version also has a typo!

Reply:
Oh! Thanks a lot sir for that. I will correct it right off the bat!